docs/hello.rst

```diff
@@ -91,7 +93,9 @@ Taichi code can run on CPUs or GPUs. Initialize Taichi according to your hardwar
 
     (OK: supported, WIP: work in progress, N/A: not available)
 
-    If the machine does not have CUDA support, Taichi will fall back to CPUs instead.
+    When specified ``arch=ti.gpu``, Taichi will try to run on CUDA.
+    If CUDA is not supported or not installed on your machine, it will fallback to Metal or OpenGL.
+    If Metal or OpenGL is also not supported, it will fallback to CPUs.
```

Comment:
`If Metal or OpenGL is also not supported` -> `If no GPU backend (CUDA, Metal, or OpenGL) is supported`